### PR TITLE
[IPU] fix ut timeout 2

### PR DESCRIPTION
**Pull request by gglin001**
### PR types

Others
### PR changes

Others
### Describe

releated PR https://github.com/PaddlePaddle/Paddle/pull/45181
fix timeout for IPU UTs when using a physical IPU

### CODE REVIEW

File: python/paddle/fluid/tests/unittests/ipu/op_test_ipu.py [hunk 1]
```diff
@@ -204,6 +204,14 @@ def run_op_test(self, exec_mode, ipu_strategy=None):
             if self.is_fp16_mode(exec_mode):
                 ipu_strategy.set_precision_config(enable_fp16=True)
                 IPUOpTest.cast_model_to_fp16(self.main_prog)
+
+            # TODO(ipu) remove in the future version of popart
+            # keep the log clean, no side effects for tests without profiling
+            ipu_strategy.set_options(
```
Comment (gglin001) [inline python/paddle/fluid/tests/unittests/ipu/op_test_ipu.py:210]: from 
https://github.com/graphcore/popart/blob/e1c072ef5aa30168d968a5648e1e62e3e0a2f9af/willow/src/engineoptionscreator.cpp#L28

这里不能完全消除所有的 warn, 部分UT不走这里

**Review by qili93 — APPROVED** [review]
LGTM

**Review by XieYunshen — APPROVED** [review]
LGTM
单测时间设置

**Comment by gglin001** [comment]
@qili93 hi, 如果没有更多的问题, 麻烦帮忙合并下这个PR, 感谢

**Review by qili93 — APPROVED** [review]
LGTM

**Pull request merged**